Scripts/framework-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Gets or sets framework version.
+# Usage: framework-version.sh               - to get the current version
+#        framework-version.sh <version>     - to set the version
+
+PROJECT_DIR="$(dirname "$0")/.."
+VERSION_CONFIG="$PROJECT_DIR/Config/Version.xcconfig"
```

Comment:
Wrong path

Reply:
What should the path be in this case?

Reply:
The current location of this file here is `"$PROJECT_DIR/WindowsAzureMessaging/WindowsAzureMessaging/Config/Version.xcconfig"`